nextcord/threads.py

```diff
@@ -143,6 +150,7 @@ class Thread(Messageable, Hashable):
         'archiver_id',
         'auto_archive_duration',
         'archive_timestamp',
+        "create_timestamp"
```

Comment:
Please change the quotes to `'` for consistency, and add a trailing comma.

Reply:
~~double quotes are better tho~~ Done!